# 'hatch-fancy-pypi-readme' is not really necessary to build 
diff --git a/pyproject.toml b/pyproject.toml
index 06e6cc88252..5628a8df080 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["hatchling>=1.20.0", "hatch-vcs", "hatch-fancy-pypi-readme"]
+requires = ["hatchling>=1.20.0", "hatch-vcs"]
 build-backend = "hatchling.build"
 
 [project]
@@ -53,12 +53,6 @@ test = [
 [project.urls]
 Homepage = "https://github.com/wkentaro/gdown"
 
-[tool.hatch.metadata.hooks.fancy-pypi-readme]
-content-type = "text/markdown"
-fragments = [
-  { path = "README.md" },
-]
-
 [tool.hatch.version]
 source = "vcs"
 
